wp-content/civi-extensions/goonjcustom/api/v3/Goonjcustom/MonthlySummaryForDroppingCenterCron.php

```
@@ -0,0 +1,87 @@
+ * @param array $params
+ *
+ * @return array API result descriptor
+ *
+ * @see civicrm_api3_create_success
+ * @see civicrm_api3_create_error
+ *
+ * @throws \CRM_Core_Exception
+ */
+function civicrm_api3_goonjcustom_monthly_summary_for_dropping_center_cron($params) {
+  $returnValues = [];
+  $today = new \DateTime();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$today'. (undefined)

(UnusedLocalVariable)